src/main/webapp/WEB-INF/tags/instructor/results/filterPanel.tag

```diff
@@ -81,7 +81,7 @@
                                         </option>
                                     </c:forEach>
                                     <option value="None"<c:if test="${filterPanel.noneSectionSelected}"> selected</c:if>>
-                                        Not in a section
+                                        No specified recipient
```

Comment:
Rather than having to repeat, let's just call `Const.USER_NOT_IN_A_SECTION` here.